### ISSUE TITLE
How to plot the loss function plot and interpret these black box models?

### ISSUE THREAD

**Issue by Jwenyi** [RaphaelS1/survivalmodels#21]
Hi Raphael,
I'm a clinician and very interested in your packages and I had read your tutorial entitled "Neural Networks for Survival Analysis in R". However, I still had some questions:
1) How to plot the loss function plot? It's necessary for publication.
2) It has any way to interpret models, such as DeepHit or DeepSurv? I used to traning a XGBoost via 'xgboost' R package and use SHAP value to interpret my own XGBoost with R package 'SHAPforxgboost', it really helpful for us to understand the decision process of black box models. However, do SHAPvalue or something like variables importance could be applied for interpretation?

**Comment by RaphaelS1** [comment]
> How to plot the loss function plot? It's necessary for publication.

I don't know what plot you're talking about sorry

> It has any way to interpret models, such as DeepHit or DeepSurv? I used to traning a XGBoost via 'xgboost' R package and use SHAP value to interpret my own XGBoost with R package 'SHAPforxgboost', it really helpful for us to understand the decision process of black box models. However, do SHAPvalue or something like variables importance could be applied for interpretation?

See this discussion https://github.com/RaphaelS1/survivalmodels/issues/9 
Short answer is that this is possible but I haven't had a chance to do this yet
